docs/provisioning_guide/src/asciidoc/_chapters/introduction.adoc

```diff
@@ -0,0 +1,466 @@
+and so on. The upgrade tasks vary depending on the differences between the current and new release of
+Trafodion. For example, an upgrade may or may not include an upgrade of the Trafodion metadata.
+
+* *<<activate,Activate>>*: Activities related to starting the Trafodion software. These actives
+include basic management tasks such as starting and checking the status of the Trafodion components,
+performing basic smoke tests, and so forth.
+
+* *<<remove,Remove>>*: Activities related to removing Trafodion from your Hadoop cluster.
+
+[[introduction-provisioning-master-node]]
+== Provisioning Master Node
+All provisioning tasks are performed from a single node in the cluster, which must be part
+of the Hadoop environment you're adding Trafodion to. This node is referred to as the
+"*Provisioning Master Node*" in this guide.
+
+The Trafodion Provisioning User must have access to all other nodes from the Provisioning
+Master Node in order to perform provisioning tasks on the cluster.
+
+[[introduction-trafodion-installer]]
+== Trafodion Installer
+
+The Trafodion Installer is a set of scripts automates most of the tasks requires to install/upgrade Trafodion.
+You download the Trafodion Installer tar file from the Trafodion http://trafodion.apache.org/download.html[download] page.
+Next, you unpack the tar file.
+
+*Example*
+
+```
+$ mkdir $HOME/trafodion-installer
+$ cd $HOME/trafodion-downloads
+$ tar -zxf apache-trafodion-installer-1.3.0-incubating-bin.tar.gz -C $HOME/trafodion-installer
+$ ls $HOME/trafodion-installer/installer
+bashrc_default           tools                             traf_config_check           trafodion_apache_hadoop_install  traf_package_setup
+build-version-1.3.0.txt  traf_add_user                     traf_config_setup           trafodion_config_default         traf_setup
+dcs_installer            traf_apache_hadoop_config_setup   traf_create_systemdefaults  trafodion_install                traf_sqconfig
+rest_installer           traf_authentication_conf_default  traf_getHadoopNodes         trafodion_license                traf_start
+setup_known_hosts.exp    traf_cloudera_mods98              traf_hortonworks_mods98     trafodion_uninstaller
+$ 
+```
+
+<<<
+The Trafodion Installer supports two different modes:
+
+1. *Guided Setup*: Prompts for information as it works through the installation/upgrade process. This mode is recommended for new users.
+2. *Automated Setup*: Required information is provided in a pre-formatted bash-script configuration file, which is provided
+via a command argument when running the Trafodion Installer thereby suppressing all prompts.
++
+A template of the configuration file is available here within the installer directory: `trafodion_config_default`.
+Make a copy of the file in your directory and populate the needed information.
++
+Automated Setup is recommended since it allows you to record the required provisioning information information ahead of time.
```

Comment:
Too bad the "Guided Setup" doesn't create a config file with the responses to all the prompts.